### DISCUSSION TITLE
Traefik and a second Fief Workspace

### DISCUSSION THREAD

**Discussion by ClaudioBsh** [fief-dev/fief#330] [Q&A] [answered]
I have a default Workspace fief.mydomain.com which is configured as a traefik host rule in the labels section of my service fief-server. 

When i call fief.mydomain.com/admin i can work with it.

I have created a second Workspace called fieftest.mydomain.com, but when i call it, i get an http code 526. Sure there is no traefik host rule label defined for this url.

So i wonder, how to solve this?

Do I have to create one more fief-server service, e.g. fief-server-test in my docker-compose.yml so i can give them an own traefik host rule label?

**Comment by frankie567** [comment, marked as answer]
An error 526 is an SSL/HTTPS related error, so there is probably something wrong with your Traefik configuration.

Either way, you should definitely add labels so Traefik can route both domains to the Fief container.

---

Please note that we're currently working on removing multiple workspaces support (see: https://github.com/orgs/fief-dev/discussions/322). So, in the next version, you'll indeed need one container per process.
